src/flow/darcy_flow_mh.hh

```diff
@@ -195,16 +162,20 @@ public:
         arma::vec4 gravity_;
         arma::vec3 gravity_vec_;
 
+        // Mirroring the following members of DarcyMH:
         Mesh *mesh;
         MH_DofHandler *mh_dh;
 
-        RichardsSystem system_;
-        uint water_balance_idx_;
-
-
-        //FieldSet  time_term_fields;
-        //FieldSet  main_matrix_fields;
-        //FieldSet  rhs_fields;
+        uint water_balance_idx;
+        std::shared_ptr<Balance> balance;
+        LinSys *lin_sys;
+        
+        unsigned int n_schur_compls;
```

Comment:
At the moment, n_schur_compls and water_balance_idx are copied from DarcyMH members into EqData..
We need these (and some others..) in DarcyAssembler, so we pass them through EqData.
I am not sure where these are supposed to be saved and how they are supposed to be passed, since these do not have much in common with EqData..